website_snippet_marginless_gallery/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2015-2017 Tecnativa - Jairo Llopis <jairo.llopis@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+{
+    "name": "Marginless Gallery Snippet",
+    "summary": "Add a snippet to have a marginless image gallery",
+    "version": "12.0.1.0.0",
+    "category": "Website",
+    "license": "LGPL-3",
+    "website": "https://odoo-community.org/project/website-maintainers-79",
```

Comment:
@cristinamartinrod URL should be `https://github.com/OCA/website`

Reply:
Right. Done!